versions/v5/functions/display-logic.php

```diff
@@ -0,0 +1,231 @@
+<?php
+/**
+ * Displays a single story on the front page.
+ **/
+function display_front_page_story( $story, $css_class='', $show_vertical=false, $thumbnail_size='frontpage-story-thumbnail', $heading='h3' ) {
+	if ( !$story ) { return false; }
```

Comment:
Curly brackets can be removed. They're not necessary when the one-line statement is on the same line as the conditional.

Reply:
Updated in both `display-logic.php` files.